os/getting-started-with-systemd.md

```diff
@@ -60,7 +60,7 @@ Feb 11 17:46:28 localhost docker[23470]: Hello World
 <a class="btn btn-default" href="{{site.baseurl}}/docs/launching-containers/launching/overview-of-systemctl">Overview of systemctl</a>
```

Comment:
lose the `{{site.baseurl}}`

these links are out of date, but the redirects should work just fine